data/metric/source_type.go

```diff
@@ -14,7 +14,9 @@ type SourceType int
 // If any more SourceTypes are added update maps: SourcesTypeToName & SourcesNameToType.
 const (
 	// GAUGE is a value that may increase and decrease. It is stored as-is.
-	GAUGE SourceType = iota
+	GAUGE   SourceType = iota
+	COUNT   SourceType = iota
+	SUMMARY SourceType = iota
 	// RATE is an ever-growing value which might be reset. The package calculates the change rate.
 	RATE SourceType = iota
```

Comment:
you can remove old types but pdelta.

Reply:
i was going to ask which ones are we still supporting. 
pdelta means we have to "store" the metric and do the delta calculation? (i removed all that code)